docs/sources/sharing/share-panel.md

```diff
@@ -57,6 +61,8 @@ If you created a snapshot by mistake, click **delete snapshot** to remove the sn
 
 You can embed a panel using an iframe on another web site. Unless anonymous access permission is enabled, the viewer must be signed into Grafana to view the graph.
 
+{{< docs-imagebox img="/img/docs/sharing/Share_Panel_Embedded_Link.png" max-width="700px" >}}
```

Comment:
See earlier comments about image link and file name.

Reply:
Will fix in separate PR.